pkg/pods/pod.go

```diff
@@ -360,6 +360,10 @@ func setupConfig(envDir string, configDir string, podManifest *PodManifest) erro
 	if err != nil {
 		return err
 	}
+	err = writeEnvFile(envDir, "POD_HOME", PodPath(podManifest.ID()))
```

Comment:
This is not guaranteed to be correct. `PodPath` provides a default pod path if you only have the manifest ID. You should make `setupConfig` a member function on `Pod` and then use `pod.Path()` instead.